apps/infrastructure/src/alb.tf

```diff
@@ -90,4 +90,33 @@ resource "aws_alb_target_group" "nextjs" {
     path                = "/"
     unhealthy_threshold = "2"
   }
+}
+
+resource "aws_lb_listener_certificate" "lb_cms_certificate" {
+  for_each = {
+    for key, config in var.websites_configs :
+    key => config
+  }
+  listener_arn    = aws_lb_listener.front_end_https.arn
+  certificate_arn = module.cms_multitenant_ssl_certificate[each.key].acm_certificate_arn
+}
+
+resource "aws_lb_listener_rule" "cms_rule" {
+  for_each = {
+    for key, config in var.websites_configs :
+    key => config
+  }
+  listener_arn = aws_lb_listener.front_end_https.arn
+  priority     = 100
```

Comment:
I think priorities cannot overlap. For this reason, it is possible to unset it and terraform will automatically populate the field.

Reply:
ok. "[priority](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lb_listener_rule#priority) - (Optional) The priority for the rule between 1 and 50000. Leaving it unset will automatically set the rule with next available priority after currently existing highest rule. A listener can't have multiple rules with the same priority.

Reply:
with last commit i have removed the rule priority